tests/ut_textchannellistener/ut_textchannellistener.cpp

```diff
@@ -129,8 +129,11 @@ void Ut_TextChannelListener::cleanupTestCase()
         model.getGroups(IM_ACCOUNT_PATH);
         QVERIFY(waitSignal(modelReady, 5000));
 
-        while (model.rowCount() != 0)
+        while (model.rowCount() != 0) {
+            int waitForDelete = model.rowCount()-1;
             QVERIFY(model.deleteGroups(QList<int>() << model.group(model.index(0,0)).id()));
+            QTRY_COMPARE(model.rowCount(), waitForDelete);
```

Comment:
Well I have no idea why rowCount changes with delay, but without waiting for the delayed change this while loop will hang in busyloop and test will fail after testrunner-lite kills it after few mins.